apps/wallet/src/ui/app/pages/home/transfer-coin/TransferCoinForm/StepOne.tsx

```diff
@@ -88,12 +78,11 @@ function StepOne({
                         className={st.input}
                         decimals
                     />
-
-                    <ErrorMessage
-                        className={st.error}
-                        name="amount"
-                        component="div"
-                    />
+                    {errors['amount'] && touched['amount'] ? (
```

Comment:
bracket access isn't necessary here (reminds me we should add a lint rule for consistent property access)